fix(Storage): iOS storage operations cancel/pause/resume (#3290)
aws-amplify/docs: docs/lib/storage/fragments/ios/upload.md

</amplify-block>

</amplify-block-switcher>

## Cancel, Pause, Resume

> After you call `uploadFile` as above, your containing class retains a reference to the operation that is actually performing the upload.

Reviewer: ```suggestion
Calls to `uploadData` or `uploadFile` return a reference to the operation that is actually performing the upload.
```

Reviewer: Same recommendation as download, except that I added uploadData for completeness and to be consistent with the download documentation.
